tools/mo/openvino/tools/mo/utils/ir_reader/extenders/sqrt_extender.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from openvino.tools.mo.front.common.partial_infer.eltwise import eltwise_infer
+from openvino.tools.mo.front.common.partial_infer.utils import float32_array
+from openvino.tools.mo.utils.graph import Node
+from openvino.tools.mo.utils.ir_reader.extender import Extender
+
+
+class Sqrt_extender(Extender):
+    op = 'Sqrt'
+
+    @staticmethod
+    def operation(a):   # we use same function as for elementwise.power but with fixed power = 0.5
+        if np.issubdtype(a.dtype, np.signedinteger):
+            return float32_array(a.astype(np.float32) ** 0.5)
+        return a ** 0.5
+
+    @staticmethod
+    def extend(op: Node):
+        op['infer'] = lambda node: eltwise_infer(node, Sqrt_extender.operation)
```

Comment:
let us add `Sqrt` shape and value infer propagation in `ops` directory please.
It is useful routine and can become useful in future.

Reply:
Done.